fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/MergeTwoAggRule.java

```diff
@@ -112,7 +112,9 @@ public boolean check(OptExpression input, OptimizerContext context) {
 
             CallOperator newFn;
             if (aggCallMapBelow.containsKey(ref)) {
-                newFn = new CallOperator(fn.getFnName(), fn.getType(), aggCallMapBelow.get(ref).getChildren(),
+                CallOperator belowOp = aggCallMapBelow.get(ref);
+                fn.getFunction().setArgTypes(belowOp.getFunction().getArgs());
+                newFn = new CallOperator(fn.getFnName(), fn.getType(), belowOp.getChildren(),
```

Comment:
```suggestion
                CallOperator belowOp = aggCallMapBelow.get(ref);
                newFn = new CallOperator(fn.getFnName(), fn.getType(), belowOp.getChildren(), belowOp.getFunction(), fn.isDistinct());
```

Reply:
Nice work, but suggest don't modify the args type of function(it's a fixed signature, modify it maybe cause BE crash), you can use below aggregate function direct for the bug.